### ISSUE TITLE
prop-shorthand: invalid warning when getting last element in array

### ISSUE THREAD

**Issue by facboy** [wix/eslint-plugin-lodash#91]
I might be missing something, but I don't think I can convert this to shorthand even though the plugin is saying that I can:

```
var maxVal = _.max(_.map(results, function (result) {
    return result.data[result.data.length - 1][0];
}));
```

(though obviously this is not the prettiest code to begin with).  Or is there property shorthand for 'the last element in an array'.

**Comment by ganimomer** [comment]
No, you're right, the rule shouldn't report on this.
On an unrelated note, you could technically do something like
```js
var getInsertSignificantNameHere = _.flow(_.property('data'), _.last, _.head);
var maxVal = _(results).map(getInsertSignificantNameHere).max();
```
 to define that function.
